### DISCUSSION TITLE
Changing the default tab or tab order?

### DISCUSSION THREAD

**Discussion by XDM-Inc** [lllyasviel/Fooocus#3775] [General]
so i'm just trying to do a tiny little quality of life mod to fooocus, i'm trying to make it launch fooocus that "inpaint or outpaint" is the new default tab which is the third one. either that or to move the third tab to the first tab. i've been tinkering around in the webui.py which i'm assuming is the correct place to start maybe. but so far i'm just having a little trouble figuring out which blocks of code start and end other lines of code. i tried a couple of things already but i got a lot of syntax errors obviously for not knowing where things end and start. any insights?

**Comment by mashb1t** [comment]
You can just move the Gradio code block for the tab using a code editor, but keep in mind not to change any indent.

**Reply by XDM-Inc** [reply]
Do I have to use a gradio tool or just carefully by hand?

**Comment by XDM-Inc** [comment]
found out how. the config file allows for a call from the webUI.py for "inpaint_tab"
by adding the line
`  "default_selected_image_input_tab_id": "inpaint_tab",`
it now does exactly what i want and launches on inpaint
